distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Case_Sensitivity.enso

```diff
@@ -25,3 +27,11 @@ type Case_Sensitivity
         Case_Sensitivity.Sensitive -> TextFoldingStrategy.unicodeNormalizedFold
         Case_Sensitivity.Insensitive locale ->
             TextFoldingStrategy.caseInsensitiveFold locale.java_locale
+
+    ## PRIVATE
+       Is case insensitive.
+    is_case_insensitive : Boolean
+    is_case_insensitive self = case self of
+        Case_Sensitivity.Default -> False
+        Case_Sensitivity.Sensitive -> False
+        Case_Sensitivity.Insensitive _ -> True
```

Comment:
I'm wondering if this method should be here/be called this way.

`Case_Sensitivity.Default` is not necessarily always `case insensitive = False` - this depends on the context where this is used. This method suggests that the default is always sensitive, which is not the case.

I would rename the method to something else, maybe `is_case_insensitive_by_default` or `is_case_insensitive_for_in_memory`? Not really great names, I know.

Maybe this method should be moved elsewhere? - `Case_Sensitivity` is an interface used in multiple places (in-memory, DB) where the defaults have different meanings depending on context. And this `is_case_insensitive` is actually an implementation detail of the in-memory backend.

If we do not want to change the method name nor its place, I would add least add a mention in the docstring explaining that this is a check meant for in-memory only, to avoid confusion.

Reply:
will add to my PR